ui/pages/settings/index.scss

```diff
@@ -111,6 +111,10 @@
             color: var(--color-icon-default);
           }
 
+          &__highlight {
+            background-color: var(--brand-colors-yellow-yellow500);
```

Comment:
Sounds like we are good to go with warning muted here. See https://consensys.slack.com/archives/C02VD8QG0LT/p1708995098553229?thread_ts=1708994247.278359&cid=C02VD8QG0LT

```suggestion
            background-color: var(--color-warning-muted);
```

Reply:
muted lgtm too
<img width="786" alt="Screenshot 2024-02-27 at 7 38 35 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/a92255b8-5f1e-4dec-8b1b-789fc9bb9ea3">
<img width="762" alt="Screenshot 2024-02-27 at 7 39 38 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/7046dfaf-3b01-4edc-90f1-fcec539b47c9">

Reply:
if we wanted to keep the yellow, we could add a text color while the [highlight darkmode](https://github.com/MetaMask/metamask-extension/issues/22647) issue is in progress

#### highlight before added text color
<img width="764" alt="Screenshot 2024-02-27 at 7 34 06 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/9c088497-6724-4232-a0b0-fff98bb8182f">

#### highlight with added text color 
```color: var(--brand-colors-grey-grey600)```
<img width="769" alt="Screenshot 2024-02-27 at 7 37 12 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/4214cf9d-bc6f-4fbe-acbf-563f8a231c8b">

Reply:
It's too light

<img width="1119" alt="Screenshot 2024-02-27 at 8 55 35 AM" src="https://github.com/MetaMask/metamask-extension/assets/26469696/bafe2051-6130-4c78-87a0-0e8114bdc021">
<img width="793" alt="Screenshot 2024-02-27 at 8 55 21 AM" src="https://github.com/MetaMask/metamask-extension/assets/26469696/52ac3e12-683e-421d-8714-30104d489908">

Reply:
Using warning inverse for text color and warning default for background 
<img width="768" alt="Screenshot 2024-02-27 at 10 15 29 AM" src="https://github.com/MetaMask/metamask-extension/assets/26469696/88f22f8c-a072-4731-9d98-95c870182130">
<img width="676" alt="Screenshot 2024-02-27 at 10 15 16 AM" src="https://github.com/MetaMask/metamask-extension/assets/26469696/bad71dd5-8436-483c-bc16-ad0502ff8aca">

Reply:
oh right. that's also a good point. The contrast is better for a11y too